app-libs/stf/src/getter.rs

```diff
@@ -171,7 +171,7 @@ impl ExecuteGetter for TrustedGetterSigned {
 				let info = System::account(&who);
 				debug!("TrustedGetter free_balance");
 				debug!("AccountInfo for {} is {:?}", account_id_to_string(&who), info);
-				debug!("Account free balance is {}", info.data.free);
+				std::println!("⣿STF⣿ 🔍 TrustedGetter query: free balance for ⣿⣿⣿ is ⣿⣿⣿",);
```

Comment:
Haha, regarding privacy, we should remove all these logs for a production setup. Should be compiler flagged in the future I guess.

Reply:
absolutely. All logs must disappear even if we don't print balances and accounts there is too much info leak. but the println can stay as is IMO